garage/tf/models/gaussian_mlp_model.py

```diff
@@ -0,0 +1,206 @@
+"""GaussianMLPModel."""
```

Comment:
Can you should how this would be used in GaussianMLPPolicy?

Reply:
Agreed, that would be great.